applications/ColossalQA/data/tests/test.md

```diff
@@ -0,0 +1,78 @@
+# README Format File for Testing
+![Alt text](./examples/diagram.png?raw=true "Fig.1. design of the document retrieval conversation system")
+
+## Table of Contents
+
+- [Table of Contents](#table-of-contents)
+- [Install](#install)
+- [How to Use](#how-to-use)
+- Examples
+  - [Local Chinese Retrieval QA + Chat](examples/retrieval_conversation_zh.py)
+  - [Local English Retrieval QA + Chat](examples/retrieval_conversation_en.py)
+  - [Local Bi-lingual Retrieval QA + Chat](examples/retrieval_conversation_universal.py)
+  - [Experimental AI Agent Based on Chatgpt + Chat](examples/conversation_agent_chatgpt.py)
+
+**As Colossal-AI is undergoing some major updates, this project will be actively maintained to stay in line with the Colossal-AI project.**
```

Comment:
Why do we move this markdown here in the tests folder.

Reply:
this is only a markdown format file for testing the data loader module as indicated in line 1